internal/venice-common/src/main/java/com/linkedin/venice/views/VeniceView.java

```diff
@@ -106,7 +107,63 @@ public static int parseVersionFromViewTopic(String topicName) {
       throw new VeniceException("Cannot parse version because this is not a view topic, topic name: " + topicName);
     }
     int versionStartIndex = Version.getLastIndexOfVersionSeparator(topicName) + Version.VERSION_SEPARATOR.length();
-    int versionEndIndex = versionStartIndex + topicName.substring(versionStartIndex).indexOf(VIEW_TOPIC_SEPARATOR);
+    int versionEndIndex = versionStartIndex + topicName.substring(versionStartIndex).indexOf(VIEW_NAME_SEPARATOR);
     return Integer.parseInt(topicName.substring(versionStartIndex, versionEndIndex));
   }
+
+  /**
+   * @param topicName for the view topic
+   * @return the corresponding store name and view name for the given view topic in a single string.
+   */
+  public static String parseStoreAndViewFromViewTopic(String topicName) {
```

Comment:
Can you share some example of `viewTopic` in the javadoc to make it easier to understand the logic?